### ISSUE TITLE
Ability to delay response

### ISSUE THREAD

**Issue by ryanzec** [callmehiphop/backend#7]
I know this library is based off of angular's `$httpBackend` but one thing I hated with that library was it was not easy to delay responses if you wanted to.  I have a modified version of this library that I can create a pull request for but before that I want to make sure your good with the functionality and API of doing this.  To delay a response in my modified version of the library, the code looks like this:

```javascript
backend
.when('GET', '/api/users/1')
.options({
    delay: 2000
})
.respond({
    name: 'Jake',
    species: 'Dog',
    magicPowers: true,
    location: 'Ooooo'
}, {
    'content-type': 'application/json'
});
```

Do you like this functionality and API for enabling it?  If so I will create a pull request.

**Comment by callmehiphop** [comment]
I think that looks awesome! I'd gladly accept a PR for that. :)

**Comment by ryanzec** [comment]
added the pull request : https://github.com/callmehiphop/backend/pull/8

**Comment by callmehiphop** [comment]
Closed by #8
